pkg/store/postings_codec.go

```diff
@@ -82,27 +84,53 @@ func diffVarintEncodeNoHeader(p index.Postings, length int) ([]byte, error) {
 	return buf.B, nil
 }
 
-func diffVarintSnappyDecode(input []byte) (index.Postings, error) {
+var snappyDecodePool sync.Pool
```

Comment:
What would be a good initial buffer size? It can be either very small or big so I'm not sure if this is a good idea thus the code is written like it is now.

Reply:
Ack! Makes sense! Thanks!